### PR TITLE
Added new property and delete APIs

### PR DESCRIPTION
**Pull request by michaelrm97**
Added new isValid property and new delete methods in response to updated behavior for handling deleted service principals.

### CODE REVIEW

File: api-reference/beta/api/mobileappmanagementpolicies-delete.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+doc_type: apiPageType
+---
+
+# Delete mobileAppManagementPolicy
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Delete a [mobilityManagementPolicy](../resources/mobilitymanagementpolicy.md) object. Note that this is only supported when the policy is no longer valid, that is, if that 'idValid' property is false. This indicates that the service principal associated with the application for this policy has been deleted.
```
Comment (RavennMSFT) [inline api-reference/beta/api/mobileappmanagementpolicies-delete.md:16]: Note should be highlighted in a separate line. See the other API docs on how a Note is added

[the remark this answers is not in the document]
Comment (michaelrm97) [reply api-reference/beta/api/mobileappmanagementpolicies-delete.md]: Fixed the typos

File: changelog/Microsoft.MobilityService.json [hunk 2]
```diff
@@ -41,6 +41,46 @@
                     "ChangeType": "Addition",
                     "Description": "Added new [Policy.ReadWrite.MobilityManagement](https://docs.microsoft.com/en-us/graph/permissions-reference#policy-permissions) delegated permission for policy APIs.",
                     "Target": "policyRoot"
+                },
+                {
```
Comment (Lauragra) [inline changelog/Microsoft.MobilityService.json:45]: It looks like you added new entries to an existing ChangeList item. Please create a new ChangeList item for these changes. Please also make sure that the descriptions end with a period.

File: changelog/Microsoft.MobilityService.json [hunk 3]
```diff
@@ -41,6 +41,46 @@
...
+                    "Id": "9ebdf457-11ea-493d-8dfb-ce12f80664bb",
+                    "ApiChange": "Method",
+                    "ChangedApiName": "mobilityManagementPolicy",
+                    "ChangeType": "Addition",
+                    "Description": "Added the [delete mobilityAppManagementPolicy](/graph/api/mobileappmanagementpolicies-delete?view=graph-rest-beta) method",
+                    "Target": "mobileAppManagementPolicies"
+                },
+                {
+                    "Id": "9ebdf457-11ea-493d-8dfb-ce12f80664bb",
+                    "ApiChange": "Resource",
```
Comment (Lauragra) [inline changelog/Microsoft.MobilityService.json:71]: For permission updates, the ApiChange value should be "Permission".

File: changelog/Microsoft.MobilityService.json [hunk 4]
```diff
@@ -41,6 +41,46 @@
...
+                    "ChangedApiName": "mobileDeviceManagementPolicies",
+                    "ChangeType": "Update",
+                    "Description": "Updated permission scopes necessary to call API",
```
Comment (Lauragra) [inline changelog/Microsoft.MobilityService.json:74]: Please make this more specific - i.e. "Updated the delegated permission scope to use...".

File: changelog/Microsoft.MobilityService.json (hunk 2 above)
Comment (michaelrm97) [reply changelog/Microsoft.MobilityService.json:45]: Not sure what you mean by create a new ChangeList item. All the changes are in new entries and with a new unique id. Can you please clarify?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**